src/notes/notes.service.spec.ts

```diff
@@ -45,6 +46,7 @@ describe('NotesService', () => {
   let forbiddenNoteId: string;
 
   beforeEach(async () => {
+    userRepo = new Repository<User>();
```

Comment:
Please add an explanation why you do this here?

Reply:
done